kolibri/plugins/management/assets/src/vue/user-page/user-create-modal.vue

```diff
@@ -91,13 +91,32 @@
 
 <style lang="stylus" scoped>
 
+  @require '~core-theme'
+
   $button-content-size = 1em
 
   .user-field
     padding-bottom: 5%
-    input, select
+    input
       width: 100%
+      height: 40px
+      font-weight: bold
     label
       position: relative
+      cursor: pointer
+    select
+      -webkit-appearance: menulist-button
```

Comment:
We should avoid using prefixed attributes if at all possible. Also, make sure you're testing other browsers in browserstack! ;-)